test/support/builders/models/submission/CISSubmissionBuilder.scala

```diff
@@ -20,10 +20,19 @@ import models.submission.CISSubmission
 import support.builders.models.submission.PeriodDataBuilder.aPeriodData
 
 object CISSubmissionBuilder {
-  val aCISSubmission: CISSubmission = CISSubmission(
+  val aCreateCISSubmission: CISSubmission = CISSubmission(
```

Comment:
would it better to have one val for CISSubmissionBuilder and then create vals in tests where it's used, similar to the other tests?